OpenTabletDriver.Desktop/Settings.cs

```diff
@@ -374,6 +376,32 @@ public static Settings Deserialize(FileInfo file)
                 return serializer.Deserialize<Settings>(jr);
         }
 
+        public static void Recover(FileInfo file, Settings settings)
+        {
+            using var stream = file.OpenRead();
+            using var sr = new StreamReader(stream);
+            using var jr = new JsonTextReader(sr);
```

Comment:
I don't like this format, this should instead be in a block format. Don't directly apply this suggestion, doesn't update lower text.
```suggestion
            using (var stream = file.OpenRead())
            using (var sr = new StreamReader(stream))
            using (var jr = new JsonTextReader(sr))
            {
```